src/genpy/dynamic.py

```diff
@@ -165,7 +165,8 @@ def generate_dynamic(core_type, msg_cat):
     
     # write the entire text to a file and import it (it will get deleted when tmp_dir goes - above)
     tmp_file = tempfile.NamedTemporaryFile(suffix=".py",dir=tmp_dir,delete=False)
-    tmp_file.file.write(full_text.encode())
+    #tmp_file.file.write(full_text.encode())
```

Comment:
Please remove this line since the next line replaces it. The git history will be sufficient.